### ISSUE TITLE
error on composer dev dependencies

### ISSUE THREAD

**Issue by bshaffer** [CoderKungfu/php-queue#50]
```
  Problem 1
    - Installation request for microsoft/windowsazure dev-master -> satisfiable by microsoft/windowsazure[dev-master].
    - microsoft/windowsazure dev-master requires pear-pear2.php.net/http_request2 * -> no matching package found.

```

**Comment by miccheng** [comment]
Have you tried running composer without the Development flag?

composer install --no-dev 

https://getcomposer.org/doc/03-cli.md#install

**Comment by bshaffer** [comment]
Yes, but that's a workaround. The issue is that the dev dependencies are broken. I wanted to bring this to your attention.

**Comment by miccheng** [comment]
Thanks. Will try to get round to it this weekend. Unless you'll like to do a pull request. Although I suspect the Azure backend probably will need updating since that Composer package is deprecated.

**Comment by ejegg** [comment]
I moved it out of require-dev as it's already in suggest, in https://github.com/CoderKungfu/php-queue/pull/67
